src/Microsoft.PowerShell.Commands.Utility/commands/utility/WriteProgressCmdlet.cs

```diff
@@ -17,10 +16,20 @@ public sealed class WriteProgressCommand : PSCmdlet
         /// </summary>
         [Parameter(
             Position = 0,
-            Mandatory = true,
             HelpMessageBaseName = HelpMessageBaseName,
             HelpMessageResourceId = "ActivityParameterHelpMessage")]
-        public string Activity { get; set; }
+        public string Activity
+        {
+            get
+            {
+                return _activity;
+            }
+
+            set
+            {
+                _activity = string.IsNullOrEmpty(value) ? " " : value;
```

Comment:
Instead of setting to a space, I think you can just remove the check in https://github.com/PowerShell/PowerShell/blob/4f57804f468c5ad47001708014d8989eac3043bc/src/System.Management.Automation/engine/ProgressRecord.cs#L47-L50

Reply:
Won't that cause problems when an older version remotes into a newer version? New version creates a ProgressRecord where activity is null, it gets serialized and when the old version attempts to create the ProgressRecord from the data it throws because Activity is null. See: https://github.com/PowerShell/PowerShell/blob/master/src/System.Management.Automation/engine/ProgressRecord.cs#L462

I guess we could update the "ToPSObjectForRemoting" method to handle that scenario but that won't help with custom PowerShell hosts that expect Activity to never be null.

Reply:
@MartinGC94 yeah, I forgot about remoting.  I guess in this case, perhaps add a comment on why you're doing it this way.

Reply:
Comment added.